lib/pengine/pe_output.c

```diff
@@ -101,28 +101,23 @@ pe__group_text(pcmk__output_t *out, va_list args)
 {
     long options = va_arg(args, long);
     resource_t *rsc = va_arg(args, resource_t *);
-    const char *pre_text = va_arg(args, char *);
-    char *child_text = NULL;
 
-    if (pre_text == NULL) {
-        pre_text = " ";
-    }
-
-    child_text = crm_concat(pre_text, "   ", ' ');
+    char *buf = crm_strdup_printf("Resource Group: %s", rsc->id);
 
-    fprintf(out->dest, "%sResource Group: %s", pre_text, rsc->id);
+    out->begin_list(out, buf, NULL, NULL);
```

Comment:
I wonder if it would be helpful (separate from this PR) if it were called as begin_list(out, NULL, NULL, "Resource Group: %s", rsc->id)

Reply:
Yeah that's a pretty good idea.  There's probably a couple more functions that could benefit from taking a list of varargs.  list_item immediately comes to mind.

Reply:
See: #1896.